scripts/postinstall.js

```diff
@@ -1,38 +1,37 @@
-#!/usr/bin/env node
-
-var fs     = require('fs');
-var path   = require('path');
-var mkdirp = require('mkdirp');
-
-var projectConfigDir = path.normalize(process.cwd() + '/../../config/');
-var localConfigFile   = path.normalize(__dirname + '/../config/sequelize.js');
-var projectConfigFile = path.normalize(process.cwd() + '/../../config/sequelize.js');
-
-var localRcFile   = path.normalize(__dirname + '/../config/.sequelizerc');
-var projectRcFile = path.normalize(process.cwd() + '/../../.sequelizerc');
-
-try {
-  fs.lstatSync(projectConfigFile);
-} catch (ex) {
-  //unable to stat file because it doesn't exist
-  console.log("copying " + localConfigFile + " to " + projectConfigFile);
-  mkdirp.sync(path.normalize(projectConfigDir));
-  fs.createReadStream(localConfigFile).pipe(fs.createWriteStream(projectConfigFile));
-
-  // Only try to copy the files required for cli operations if sequelize.js is being newly created.
-  try {
-       fs.lstatSync(projectRcFile);
-  } catch (ex) {
-      console.log("copying " + localRcFile + " to " + projectRcFile);
-      fs.createReadStream(localRcFile).pipe(fs.createWriteStream(projectRcFile));
-  }
-}
-
-['models', 'test/fixtures'].forEach(function(f){
-  mkdirp.sync(path.normalize(process.cwd() + '/../../' + f));
-});
-
-console.warn('Warning:');
-console.warn('For actionhero 13 or higher please make sure you link this plugin.');
-console.warn('npm run actionhero link -- --name ah-sequelize-plugin');
-console.warn('To read more about this, http://www.actionherojs.com/docs/#plugins');
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+const mkdirp = require('mkdirp')
+
+const projectConfigDir = path.normalize(process.cwd() + '/../../config/')
+const localConfigFile = path.normalize(path.join(__dirname, '/../config/sequelize.js'))
+const projectConfigFile = path.normalize(process.cwd() + '/../../config/sequelize.js')
+
+const localRcFile = path.normalize(path.join(__dirname, '/../config/.sequelizerc'))
+const projectRcFile = path.normalize(process.cwd() + '/../../.sequelizerc')
+
+try {
+  fs.lstatSync(projectConfigFile)
+} catch (ex) {
+  // unable to stat file because it doesn't exist
+  console.log('copying ' + localConfigFile + ' to ' + projectConfigFile)
+  mkdirp.sync(path.normalize(projectConfigDir))
+  fs.createReadStream(localConfigFile).pipe(fs.createWriteStream(projectConfigFile))

```

Comment:
This isn't needed any more, as linking a plugin copies config files automatically: https://github.com/actionhero/actionhero/blob/master/bin/methods/link.js#L91-L92